wincurses.c

```diff
@@ -284,8 +284,9 @@ int mvaddch(int y, int x, const chtype ch)
 
 int mvwaddch(WINDOW *win, int y, int x, const chtype ch)
 {
-	move(y, x);
-	return waddch(stdscr, ch);
+	if(wmove(win,y, x)==ERR)

```

Comment:
Looks good, the only change I would make is formatting:
```
if (wmove(win, y, x) == ERR)
```

Reply:
Format have been fixed! : ) This is my first pull request and I hope that I am doing the right thing.